metricbeat/module/docker/helper.go

```diff
@@ -52,24 +51,15 @@ func ExtractContainerName(names []string) string {
 	return strings.Trim(output, "/")
 }
 
-func BuildLabelArray(labels map[string]string) []common.MapStr {
+func BuildLabelArray(labels map[string]string) common.MapStr {
```

Comment:
Looking at the what the function does, it seems to de-dot the label names. So how about `DeDotLabels()`? I would prefer that all exported functions have godoc accompanying them. So how about adding (assuming that's the name you go with):

```
// DeDotLabels returns a new common.MapStr containing a copy of the labels
// where the dots in each label name have been changed to an underscore.
```